subsys/partition_manager/pm.yml.pgps

```diff
@@ -3,10 +3,14 @@
 pgps:
   placement:
     before: [tfm_storage, end]
+#ifdef CONFIG_PM_PARTITION_REGION_PGPS_EXTERNAL
+  region: external_flash
```

Comment:
Given that the indentation has meaning in `yml` files we need to align this `region` with the `before` above to make sure its picked up as an item in the `placement` dict. IMO it would be better (if possible) to add `{` and `}` after `placement` to avoid having to ensure that the indentation is correct.

Reply:
great suggestions -- thanks